types/redux-form/v6/tsconfig.json

```diff
@@ -9,7 +9,7 @@
         "noImplicitAny": true,
         "noImplicitThis": true,
         "strictNullChecks": true,
-        "strictFunctionTypes": true,
+        "strictFunctionTypes": false,
```

Comment:
We generally don't like going back on the strictness provided by a library - what became the issue here?